.github/workflows/macOS-tests.yml

```diff
@@ -34,6 +34,11 @@ jobs:
       cancel-in-progress: false
     steps:
       - uses: actions/checkout@v3
+      - name: Discover typos
+        run: |
+          pip3 install --upgrade pip
+          pip3 install codespell
+          codespell --count --ignore-words-list=ans,deriver,inout,packag --skip="*.js,*WordLists.swift"
```

Comment:
@cclauss can you please explain the parameters and why it should be configured that way?

Reply:
% `codespell --count`
```
./Tests/web3swiftTests/localTests/ABIElementErrorDecodingTest.swift:35: InOut ==> input, in out
./Tests/web3swiftTests/localTests/EIP1559BlockTests.swift:105: Shoult ==> Should
./Tests/web3swiftTests/localTests/ABIEncoderTest.swift:106: InOut ==> input, in out
./Sources/web3swift/Web3/Web3+Instance.swift:169: inout ==> input, in out
./Sources/web3swift/Web3/Web3+Instance.swift:171: inout ==> input, in out
./Sources/web3swift/Web3/Web3+Resolver.swift:19: inout ==> input, in out
./Sources/web3swift/Web3/Web3+Signing.swift:12: inout ==> input, in out
./Sources/web3swift/Utils/ENS/ENSRegistry.swift:68: ans ==> and
./Sources/web3swift/Utils/ENS/ENSRegistry.swift:69: ans ==> and
./Sources/web3swift/Utils/EIP/EIP681.swift:286: InOut ==> input, in out
./Sources/web3swift/Utils/EIP/EIP681.swift:342: InOut ==> input, in out
./Sources/web3swift/Utils/EIP/EIP681.swift:447: InOut ==> input, in out
./Sources/web3swift/Utils/EIP/EIP681.swift:543: InOut ==> input, in out
./Sources/web3swift/Utils/EIP/EIP681.swift:546: InOut ==> input, in out
./Sources/web3swift/HookedFunctions/Web3+Wallet.swift:30: inout ==> input, in out
./Sources/web3swift/Browser/browser.js:140: sucessful ==> successful
./Sources/web3swift/Browser/browser.js:441: Overrided ==> Overrode, Overridden
./Sources/web3swift/Browser/browser.js:442: overrided ==> overrode, overridden
./Sources/web3swift/Browser/browser.js:565: Overrided ==> Overrode, Overridden
./Sources/web3swift/Browser/browser.js:568: overrided ==> overrode, overridden
./Sources/web3swift/Browser/browser.js:1557: octect ==> octet
./Sources/web3swift/Browser/browser.js:1709: rigth ==> right
./Sources/web3swift/Browser/browser.js:3513: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:3558: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:3615: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:4154: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:4889: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:5518: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:5562: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:5636: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:6055: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:7764: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:7869: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:8316: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:8515: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:8668: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:8669: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:9239: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.js:9329: inital ==> initial
./Sources/web3swift/Browser/browser.js:13416: implimentation ==> implementation
./Sources/web3swift/Browser/browser.js:14636: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:14637: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:14638: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:14639: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:15409: interpretted ==> interpreted
./Sources/web3swift/Browser/browser.js:15692: coersion ==> coercion, conversion
./Sources/web3swift/Browser/browser.js:17348: usefull ==> useful
./Sources/web3swift/Browser/browser.js:18958: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:18960: ede ==> edge
./Sources/web3swift/Browser/browser.js:19045: overrided ==> overrode, overridden
./Sources/web3swift/Browser/browser.js:19347: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19353: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19355: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19357: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19358: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19361: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19369: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:19370: EDE ==> EDGE
./Sources/web3swift/Browser/browser.js:22161: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22193: keyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22193: keyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22194: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22198: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22202: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22358: sencond ==> second
./Sources/web3swift/Browser/browser.js:22402: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22407: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22413: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22415: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22416: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22419: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22425: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22426: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22429: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22435: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22448: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22464: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22471: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22479: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22497: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22502: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22506: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22510: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22660: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22683: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22683: keypair ==> key pair
./Sources/web3swift/Browser/browser.js:22701: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22722: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22726: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22790: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22799: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22800: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22802: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22805: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22806: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22808: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22811: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22815: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22819: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22825: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22838: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22842: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22846: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22850: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22855: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22859: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22864: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:22868: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.js:24754: succesful ==> successful
./Sources/web3swift/Browser/browser.js:25752: instuction ==> instruction
./Sources/web3swift/Browser/browser.js:25897: inculded ==> included
./Sources/web3swift/Browser/browser.js:25901: inculding ==> including
./Sources/web3swift/Browser/browser.js:26529: exmple ==> example
./Sources/web3swift/Browser/browser.js:26646: inculde ==> include
./Sources/web3swift/Browser/browser.js:27512: constuctor ==> constructor
./Sources/web3swift/Browser/browser.js:28616: regardles ==> regardless
./Sources/web3swift/Browser/browser.js:28829: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.js:28932: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.js:33463: Functon ==> Function
./Sources/web3swift/Browser/browser.js:36314: nerver ==> never
./Sources/web3swift/Browser/browser.js:42943: enviroments ==> environments
./Sources/web3swift/Browser/browser.js:42952: maddness ==> madness
./Sources/web3swift/Browser/browser.js:42959: hopfully ==> hopefully
./Sources/web3swift/Browser/browser.js:42968: enviroments ==> environments
./Sources/web3swift/Browser/browser.js:42977: maddness ==> madness
./Sources/web3swift/Browser/browser.js:42984: hopfully ==> hopefully
./Sources/web3swift/Browser/browser.js:43050: predictible ==> predictable
./Sources/web3swift/Browser/browser.js:45551: falsy ==> falsely, false
./Sources/web3swift/Browser/browser.js:47145: containg ==> containing
./Sources/web3swift/Browser/browser.js:47463: interupt ==> interrupt
./Sources/web3swift/Browser/browser.js:48040: signficantly ==> significantly
./Sources/web3swift/Browser/browser.js:48088: signficantly ==> significantly
./Sources/web3swift/Browser/browser.js:48383: keyPair ==> key pair
./Sources/web3swift/Browser/browser.js:48403: keyPair ==> key pair
./Sources/web3swift/Browser/browser.js:49730: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49731: Wih ==> With
./Sources/web3swift/Browser/browser.js:49731: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49732: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49732: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49733: Wih ==> With
./Sources/web3swift/Browser/browser.js:49733: Wih ==> With
./Sources/web3swift/Browser/browser.js:49733: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49734: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49734: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49735: Wih ==> With
./Sources/web3swift/Browser/browser.js:49735: Wih ==> With
./Sources/web3swift/Browser/browser.js:49735: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49737: Wih ==> With
./Sources/web3swift/Browser/browser.js:49738: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49742: Wih ==> With
./Sources/web3swift/Browser/browser.js:49743: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49766: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:49767: Wih ==> With
./Sources/web3swift/Browser/browser.js:49767: Wil ==> Will, Well
./Sources/web3swift/Browser/browser.js:51253: hashs ==> hashes
./Sources/web3swift/Browser/browser.js:51259: datas ==> data
./Sources/web3swift/Browser/browser.js:51259: hashs ==> hashes
./Sources/web3swift/Browser/browser.js:51260: datas ==> data
./Sources/web3swift/Browser/browser.js:51261: datas ==> data
./Sources/web3swift/Browser/browser.js:51265: datas ==> data
./Sources/web3swift/Browser/browser.js:51265: datas ==> data
./Sources/web3swift/Browser/browser.js:51266: datas ==> data
./Sources/web3swift/Browser/browser.js:51397: datas ==> data
./Sources/web3swift/Browser/browser.js:51404: datas ==> data
./Sources/web3swift/Browser/browser.js:51608: Perfoms ==> Performs
./Sources/web3swift/Browser/browser.js:52257: falsy ==> falsely, false
./Sources/web3swift/Browser/browser.js:52870: egal ==> equal
./Sources/web3swift/Browser/browser.js:53214: offest ==> offset
./Sources/web3swift/Browser/browser.js:54547: Trys ==> Tries
./Sources/web3swift/Browser/browser.js:55706: indrect ==> indirect
./Sources/web3swift/Browser/browser.js:56103: succesfull ==> successful
./Sources/web3swift/Browser/browser.js:56132: succesfull ==> successful
./Sources/web3swift/Browser/browser.js:56499: requet ==> request
./Sources/web3swift/Browser/browser.js:57246: doesnt ==> doesn't, does not
./Sources/web3swift/Browser/browser.js:58338: unecessary ==> unnecessary
./Sources/web3swift/Browser/browser.js:59072: gived ==> given, gave
./Sources/web3swift/Browser/browser.js:59434: DUBLICATE ==> DUPLICATE
./Sources/web3swift/Browser/browser.js:59734: Furter ==> Further
./Sources/web3swift/Browser/browser.js:59938: Furter ==> Further
./Sources/web3swift/Browser/browser.js:59948: ressources ==> resources
./Sources/web3swift/Browser/browser.js:59949: ressources ==> resources
./Sources/web3swift/Browser/browser.js:60023: ressources ==> resources
./Sources/web3swift/Browser/browser.js:60023: ressources ==> resources
./Sources/web3swift/Browser/browser.js:60040: Furter ==> Further
./Sources/web3swift/Browser/browser.js:60067: Furter ==> Further
./Sources/web3swift/Browser/browser.js:61700: DUBLICATE ==> DUPLICATE
./Sources/web3swift/Browser/browser.js:62422: necesary ==> necessary
./Sources/web3swift/Browser/browser.js:62691: immeditately ==> immediately
./Sources/web3swift/Browser/browser.js:63302: cant ==> can't
./Sources/web3swift/Browser/browser.js:63819: inflight ==> in-flight
./Sources/web3swift/Browser/browser.js:63824: inflight ==> in-flight
./Sources/web3swift/Browser/browser.js:63831: inflight ==> in-flight
./Sources/web3swift/Browser/browser.js:64719: inflight ==> in-flight
./Sources/web3swift/Browser/browser.js:67436: arround ==> around
./Sources/web3swift/Browser/browser.js:67889: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: octect ==> octet
./Sources/web3swift/Browser/browser.min.js:1: rigth ==> right
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:1: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:1: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:1: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:1: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:1: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:1: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:1: ede ==> edge
./Sources/web3swift/Browser/browser.min.js:1: ede ==> edge
./Sources/web3swift/Browser/browser.min.js:1: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:1: sencond ==> second
./Sources/web3swift/Browser/browser.min.js:1: candinate ==> candidate
./Sources/web3swift/Browser/browser.min.js:1: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:1: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: instuction ==> instruction
./Sources/web3swift/Browser/browser.min.js:1: inculded ==> included
./Sources/web3swift/Browser/browser.min.js:1: inculding ==> including
./Sources/web3swift/Browser/browser.min.js:1: parenthsis ==> parenthesis
./Sources/web3swift/Browser/browser.min.js:1: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.min.js:1: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.min.js:1: invlaid ==> invalid
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: Invald ==> Invalid
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:1: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:1: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:1: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:1: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:1: achived ==> achieved, archived
./Sources/web3swift/Browser/browser.min.js:1: indrect ==> indirect
./Sources/web3swift/Browser/browser.min.js:1: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:1: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:1: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:1: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:1: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: octect ==> octet
./Sources/web3swift/Browser/browser.min.js:2: rigth ==> right
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:2: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:2: Ot ==> To, Of, Or, Not
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: onlyOnce ==> only once
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:2: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:2: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:2: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:2: EDE ==> EDGE
./Sources/web3swift/Browser/browser.min.js:2: ede ==> edge
./Sources/web3swift/Browser/browser.min.js:2: ede ==> edge
./Sources/web3swift/Browser/browser.min.js:2: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:2: sencond ==> second
./Sources/web3swift/Browser/browser.min.js:2: candinate ==> candidate
./Sources/web3swift/Browser/browser.min.js:2: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:2: KeyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: instuction ==> instruction
./Sources/web3swift/Browser/browser.min.js:2: inculded ==> included
./Sources/web3swift/Browser/browser.min.js:2: inculding ==> including
./Sources/web3swift/Browser/browser.min.js:2: parenthsis ==> parenthesis
./Sources/web3swift/Browser/browser.min.js:2: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.min.js:2: hexidecimal ==> hexadecimal
./Sources/web3swift/Browser/browser.min.js:2: invlaid ==> invalid
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: Invald ==> Invalid
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ot ==> to, of, or, not
./Sources/web3swift/Browser/browser.min.js:2: Te ==> The, Be, We, To
./Sources/web3swift/Browser/browser.min.js:2: Ue ==> Use, Due
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: ue ==> use, due
./Sources/web3swift/Browser/browser.min.js:2: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:2: keyPair ==> key pair
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/web3swift/Browser/browser.min.js:2: achived ==> achieved, archived
./Sources/web3swift/Browser/browser.min.js:2: indrect ==> indirect
./Sources/web3swift/Browser/browser.min.js:2: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:2: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:2: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:2: ressources ==> resources
./Sources/web3swift/Browser/browser.min.js:2: te ==> the, be, we, to
./Sources/Core/EthereumABI/ABIElements.swift:75: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:87: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:88: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:93: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:93: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:103: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:106: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:148: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:150: InOut ==> input, in out
./Sources/Core/EthereumABI/ABIElements.swift:350: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIDecoding.swift:12: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:75: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:87: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:88: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:93: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:93: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:103: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:107: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:149: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:151: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:159: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:166: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIElements.swift:448: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIEncoding.swift:140: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:73: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:77: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:78: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:82: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:98: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:102: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:121: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:125: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:132: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:140: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:149: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:159: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:162: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:176: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:186: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:197: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:200: InOut ==> input, in out
./Sources/Web3Core/EthereumABI/ABIParsing.swift:204: InOut ==> input, in out
./Sources/Web3Core/Contract/ContractProtocol.swift:35: InOut ==> input, in out
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:10: sting ==> string
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: adresse ==> address
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: agre ==> agree
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: alle ==> all, alley
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: ane ==> and
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: annonce ==> announce
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: anormal ==> abnormal, a normal
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: bobard ==> board, bombard
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: brillant ==> brilliant
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: chambre ==> chamber
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: classe ==> class, classes
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: convexe ==> convex, convexes
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: copie ==> copy, copies, copied, copier
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: cristal ==> crystal
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: cylindre ==> cylinder
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: caler ==> caller
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: loger ==> logger, lodger, longer
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: diable ==> disable
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: couter ==> counter
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: tage ==> stage, take, tag, tagged
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: exemple ==> example
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: fle ==> file
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: fle ==> file
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sie ==> size, sigh, side
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: tre ==> tree
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: groupe ==> grouped, group
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: pard ==> part
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: inventer ==> inventor
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: langage ==> language
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: limite ==> limit
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: mesure ==> measure
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: ore ==> or
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: moteur ==> motor
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: obscur ==> obscure
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: oce ==> once, one, ore
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sie ==> size, sigh, side
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: polygone ==> polygon
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: fixe ==> fixed, fixes, fix, fixme, fixer
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sence ==> sense, since
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: proble ==> problem, probe, probably
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: pyramide ==> pyramid
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: rouge ==> rogue
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sculpter ==> sculptor, sculpture
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sergent ==> sergeant
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sie ==> size, sigh, side
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sie ==> size, sigh, side
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: sigle ==> single, sigil
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: syntaxe ==> syntax
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: syste ==> system
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: trafic ==> traffic
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: tre ==> tree
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: fle ==> file
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: tre ==> tree
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:25: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:28: normale ==> normal
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:28: operato ==> operator
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:28: tabacco ==> tobacco
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: anual ==> annual
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: autor ==> author
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: avance ==> advance
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: bu ==> by, be, but, bug, bun, bud, buy, bum
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: calle ==> called
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: cliente ==> client, clientele
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: codo ==> code
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: dilema ==> dilemma
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: doble ==> double
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: elipse ==> ellipse, eclipse
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: fase ==> faze, phase
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: fiel ==> feel, field, file, phial
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: flor ==> floor, flow
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: foto ==> photo
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: ilegal ==> illegal
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: leve ==> level, levee
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: momento ==> memento
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: nin ==> inn, min, bin, nine
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: nin ==> inn, min, bin, nine
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: oce ==> once, one, ore
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: oriente ==> oriented
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: posible ==> possible
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: profesor ==> professor
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: ser ==> set
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: serie ==> series
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: te ==> the, be, we, to
./Sources/Web3Core/KeystoreManager/BIP39+WordLists.swift:31: tre ==> tree
./Sources/Web3Core/Structure/SECP256k1.swift:89: inout ==> input, in out
./Sources/Web3Core/Structure/SECP256k1.swift:130: inout ==> input, in out
./Sources/Web3Core/Structure/SECP256k1.swift:206: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:57: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:57: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:63: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:63: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:69: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:69: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:75: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:75: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:81: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:81: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:87: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:87: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:93: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:93: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:99: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:99: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:105: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:105: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:111: inout ==> input, in out
./Sources/Web3Core/Utility/RIPEMD160+StackOveflow.swift:111: inout ==> input, in out
./Sources/Web3Core/Utility/CryptoExtension.swift:16: deriver ==> derive, driver
./Sources/Web3Core/Utility/CryptoExtension.swift:17: deriver ==> derive, driver
./Sources/Web3Core/Utility/Data+Extension.swift:36: inout ==> input, in out
693
```

Reply:
@janndriessen 
`--ignore-words-list=ans,deriver,inout,packag` Ignores these words and for example type `InOut` won't trigger errors.

`--skip="*.js,*WordLists.swift"` skips browser related code (there are some js files) and skips files like `BIP39+WordLists.swift`.

Reply:
Yes.  The first option is a list of words that we ignore and the second option is those files that we skip.

Reply:
@cclauss Based on this PR I've decided to add pre-commit hooks that will run `codespell` and `swiftlint` to prevent developers from committing changes that do not match the rules: https://github.com/web3swift-team/web3swift/pull/755

Maybe you'll be also interested in looking at it.

Reply:
https://.pre-commit.ci is also worth looking into for autoupdate and autocorrections.